Nodejs/Product/Nodejs/Workspace/ContextMenuProvider.cs

```diff
@@ -149,13 +165,30 @@ private void ExecNpmInstallNew(WorkspaceVisualNodeBase node)
                 }
             }
 
+            private async void ExecDebugAsync(WorkspaceVisualNodeBase node)
+            {
+                var workspace = node.Workspace;
+                var packageJson = PackageJsonFactory.Create(((IFileNode)node).FullPath);
+
+                //invoke debuglaunchtargetprovider on this file
+                var fileContextActions = await node.Workspace.GetFileContextActionsAsync(packageJson.Main, new[] { DebugLaunchActionContext.ContextTypeGuid });
+                if (fileContextActions.Any())
+                {
+                    var action = fileContextActions.First().FirstOrDefault();
```

Comment:
>First [](start = 52, length = 5)

What's special about the first file context action?

Reply:
Changed to single, since we only asked for a single ContextType